File: MATEMÁTICA/2170/main.py
from sys import stdin


def interest_rate(x, y):
    return 100 * (y - x) / x


def formatted_print(project, rate):
    print(f'Projeto {project + 1}:')
    print(f'Percentual dos juros da aplicacao: {rate:.2f} %')
    print()


for project, line in enumerate(stdin):
    x, y = map(float, line.split())
    rate = interest_rate(x, y)
    formatted_print(project, rate)

File: MATEMÁTICA/2170/main.rb
def interest_rate(x, y)
  100 * (y - x) / x
end

def formatted_print(project, rate)
  puts "Projeto #{project + 1}:"
  puts "Percentual dos juros da aplicacao: %.2f %%" % rate
  puts
end

$stdin.each_line.with_index do |line, project|
  x, y = line.split.map(&:to_f)
  rate = interest_rate(x, y)
  formatted_print(project, rate)
end
